packages/amesos2/src/Amesos2_TpetraMultiVecAdapter_def.hpp

```diff
@@ -559,11 +553,7 @@ namespace Amesos2 {
       // MDM-TODO Is this the right way to update MV: copying one element at a time?
       // WARNING - This is not tested currently by Tacho Amesos2_Solver_Test - tested manually by setting numVecs 1 in Solver_Test.cpp. Add param?
       auto mv_view_to_modify_2d = mv_->template getLocalView<typename KV::execution_space>();
```

Comment:
I don't see any `sync*` or `modify*` calls before you modify (on what could be host or device).  I would strongly recommend using `withLocalAccess` if you don't understand how that stuff works.  Thanks!